.github/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```

Comment:
Will we have any issues doing this daily, rather than something less frequent?

Reply:
Only that PRs will arrive (at most) daily rather than on some other schedule. With auto-approve that means they should be merged as soon as the pipeline is successful, so there shouldn't be any extra work unless a package breaks the pipeline.

Reply:
Okay.